Tasks/Barabanov/Task1m.py

```diff
@@ -0,0 +1,34 @@
+from decimal import *
```

Comment:
I can see that you have some experience of working with Python, your code is very good, but I would strongly advice against using such kind of imports (from something import *) when you're dealing with the modules which were created not by you

we will discuss this topic in detail on our lessons in some time